client/lcd/test_helpers.go

```diff
@@ -1414,6 +1414,21 @@ func getSigningInfo(t *testing.T, port string, validatorPubKey string) slashing.
 	return signingInfo
 }
 
+// ----------------------------------------------------------------------
+// ICS 23 - SlashingList
+// ----------------------------------------------------------------------
+// GET /slashing/signing_infos Get sign info of given validator
```

Comment:
`given validator` leads me to think that a Validator address is somehow supplied. It does not seem it is the case here.

Reply:
Changed the comment, thanks for catching it!